test/play_scraper_test.py

```diff
@@ -0,0 +1,80 @@
+from pathlib import Path
+from urllib.error import HTTPError, URLError
+import numpy as np
+import pytest
+from src import PlayScrapingError, PlayPageScraper
+
+ICON_HTML = """
+<img src="%s" class="T75of sHb2Xb">
+"""
+
+CATEGORY_HTML = """
+<a itemprop="genre">Example</a>
+"""
+
+SIMPLE_HTML = """
+<p>Simple paragraph.</p>
+"""
+
+ICON_SUBDIR = Path("icon_subdir")
+
+
+class TestPlayPageScraper:
+    def test_get_icon(self, test_dir, icon_dir, play_scraper):
+        rand_array = np.random.randint(0, 255, 10)
+        icon_name = "icon_com.example"
+        np.savetxt(test_dir / icon_name, rand_array, fmt="%d")
+
+        html_file = test_dir / "com.example"
```

Comment:
I would move the test file creation (html and icons) into high-level helpers to make the tests easier to read.